### ISSUE TITLE
Ability to Set configuration label and profile

### ISSUE THREAD

**Issue by cforce** [codecentric/spring-boot-admin#138]
Admin Server enables  changing of active profile and configuration label of any microservices .The profile selection list is made of an union of application packages profiles and spring cloud config server profiles . The label is freetext and must be known by user, no predefined list. Optional restart of the application to apply new profile and/or labeled configuration

**Comment by joshiste** [comment]
What? I don't understand any of this german/englisch mumbo-jumbo/kauderwelsch.

**Comment by cforce** [comment]
sry, mobile phone ;)

**Comment by joshiste** [comment]
OK now in better english. But I'm not quite sure if I understand what you want...

Do you wan't a restart-button as requested in #99? Then this is a duplicate.

Currently you can set properties via Spring Clouds extended `/env`-endpoint and there is also an button to hit the `/refresh`-endpoint reinitializing the refresh-scope. So basically you could override `spring.profiles.active`.
Be aware that the changes to your properties is done using a high-priority PropertySource in spring-cloud and is **not** persisted. So restarting your application will lose the "overrides". And for changing the configuration label this has to be done very early in the bootsrap-phase... So this will never workout I think.

So basically you some-how need to change your config-files or system-environment in an durable fashion. This is totally out of scope for Spring Boot Admin. In essence Spring Boot Admin is just a _simple_ UI for Spring Boot Acutator/Spring Cloud-Endpoints. No other magic involved. It just helps with finding all your apps in a list by discovery or http-registration.
